helpers/make_network_dirs_main.py

```diff
@@ -0,0 +1,118 @@
+"""Script that creates all directories"""
+import os
+import sys
+sys.path.append("D:/research-and-development")
+from src.utils.helpers import tree_to_list
+from src.utils.local_file_mods import rd_file_exists
+
+# Change to the project repository location
+my_wd = os.getcwd()
+my_repo = "R:\\"
```

Comment:
Would it make more sense to move "root" (line 37) here, and then call the "tree_to_list" function with the default empty string for "prefix"?

Reply:
I have addressed this comment. Thank you.